packages/utilities/src/makeDraggable/makeDraggable.ts

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright IBM Corp. 2025, 2025
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+interface DraggableProps {
+  //The HTML element to move.
+  el: HTMLElement;
+  //Optional HTML element to initiate the drag (e.g., header).
+  dragHandle?: HTMLElement;
+  //Optional HTML element to focus on drag for keyboard interaction (e.g., Drag Icon).
+  focusableDragHandle?: HTMLElement;
+  //Optional pixel value that defines the distance to move when dragging with arrow keys. (default:8px)
+  dragStep?: number;
+  //Optional pixel value that defines the distance to move when dragging with shift+arrow keys. (default:32px)
```

Comment:
Three suggestions:

- Remove the 'Optional' keywords from the property descriptions, since they are redundant.
- Remove the default value references. If the defaults change in the code, the comments would also need to be updated.
- Convert the comments to JSDoc format so that IntelliSense can recognize them.

```suggestion
  /**
   * HTML element to move.
   */
  el: HTMLElement;
  /**
   * HTML element to initiate the drag (e.g., header).
   */
  dragHandle?: HTMLElement;
  /**
   * HTML element to focus on drag for keyboard interaction (e.g., Drag Icon).
   */
  focusableDragHandle?: HTMLElement;
  /**
   * Pixel value that defines the distance to move when dragging with arrow
   * keys.
   */
  dragStep?: number;
  /**
   * Pixel value that defines the distance to move when dragging with
   * shift+arrow keys.
   */
```